.travis.yml

```diff
@@ -2,6 +2,9 @@ language: java
 
 sudo: false
 
+before_install:
+  - export M2_HOME=/usr/local/maven-3.3.9
+
```

Comment:
I don't think we need this.
https://github.com/checkstyle/contribution/blob/master/.travis.yml doesn't have this at all.